src/webui/www/private/scripts/client.js

```diff
@@ -31,6 +31,7 @@ window.qBittorrent.Client ??= (() => {
         return {
             closeWindows: closeWindows,
             genHash: genHash,
+            getHost: getHost,
```

Comment:
This function is more related to `qBittorrent.Misc` than `qBittorrent.Client`. Do you mind move it there? I would also move `genHash()` too.